examples/cfd/09_Darcy_flow_equation.ipynb

```diff
@@ -0,0 +1,537 @@
+{
```

Comment:
<p>Remove print statement to avoid CI issue if a decimal doesn't match</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1998/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>